test/integration/scheduler/preemption_test.go

```diff
@@ -704,6 +705,17 @@ var _ = ginkgo.Describe("Preemption", func() {
 				Obj()
 			gomega.Expect(k8sClient.Create(ctx, aStandardVeryHighWl)).To(gomega.Succeed())
 
+			conditionCmpOpts := cmpopts.IgnoreFields(metav1.Condition{}, "LastTransitionTime", "ObservedGeneration")
+			gomega.Eventually(func(g gomega.Gomega) {
```

Comment:
Since we have so many similar `Eventually`s, please add a method `ExpectPreemptedCondition` to `test/utils`